host/traceh.c

```diff
@@ -208,11 +194,10 @@ void oe_log_message(bool is_enclave, oe_log_level_t level, const char* message)
     // Take the log file lock.
     if (oe_mutex_lock(&_log_lock) == OE_OK)
     {
-        if (!_log_file_name)
-        {
-            _write_message_to_stream(stdout, is_enclave, level, message);
-        }
-        else if (!_log_creation_failed_before)
+        _write_message_to_stream(
```

Comment:
What is the rationale for always printing the log to stdout?

Reply:
I am making the assumption that a user would still like the logs to be outputted as normal, but also optionally redirect them to a file in a machine-readable format to be picked up by a log analytics pipeline. We can discuss this assumption further if people are uncomfortable with it.

Reply:
That takes away the option of logging to file only.
I recommend not to spew anything out when it's set to file

Reply:
Now by default when a file is specified it will log only to the file. Added another env. variable that if set enables the option to log to stdout and to the log file.

Reply:
Thanks for addressing my comment